<launch>
    <arg name="sensor_id"      default="20000"   doc="Sensor ID to process (e.g., 1)" />
    <arg name="start_index"    default="0"   doc="Start index of the data sequence" />
    <arg name="load_count"     default="10"  doc="Number of frames to process" />
    <arg name="output_seq_id"  default="10"  doc="Sequence ID for the output bin file (Avoids conflict with map data 0~3)" />
    <arg name="package_name"   default="toy_map_viewer" />

    <node pkg="toy_map_viewer" type="coordinate_converter_node" name="coordinate_converter_node" output="screen">
        <param name="package_name"  value="$(arg package_name)" />
        <param name="sensor_id"     value="$(arg sensor_id)" />
        <param name="start_index"   value="$(arg start_index)" />
        <param name="load_count"    value="$(arg load_count)" />
        
        <param name="output_seq_id" value="$(arg output_seq_id)" />
    </node>
</launch>